### PR TITLE
Move to copier #1: Basic port

### PR DESCRIPTION
**Pull request by ruestefa**
Only the necessary changes to get the blueprint running with copier:

- Remove obsolete files in repo root
- Replace `cookiecutter.json` by `copier.yaml`
  - Add parameter `year` b/c there's no `now()` in pure jinja2
- Rename `{{cookiecutter.project_slug}}/` to `tmpl/`
- Remove `cookiecutter.` prefix from all jinja2 templates

### CODE REVIEW

**Comment by ruestefa** [comment]
> This branch has conflicts that must be resolved
> to resolve conflicts before continuing.
> Conflicting files
> `{{cookiecutter.project_slug}}/pyproject.toml`
> `{{cookiecutter.project_slug}}/run-mypy.sh`
> `{{cookiecutter.project_slug}}/test_all.sh`

Conflicts are caused by the fact that the files were both renamed (`{{cookiecutter.project_slug}}/ -> tmpl/`) and changed (removed `cookiecutter.` prefix from jinja2 templates). The incoming changes from `copier1_port` can be accepted w/o adaptation during the merge.

**Comment by ruestefa** [comment]
Add intermediate branch `copier0b_rename` (#53) to prevent merge conflicts.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -6,33 +6,33 @@ repos:
...
-    hooks:
-    -   id: isort
-        files: ^tests/([^/]+/)*\w+.py$
--   repo: https://github.com/pycqa/flake8
-    rev: 5.0.4
-    hooks:
-    -   id: flake8
-        files: ^tests/([^/]+/)*\w+.py$
-        args: [--max-line-length=88]
+# -   repo: https://github.com/psf/black
```
Comment (ruestefa) [inline .pre-commit-config.yaml:23]: Remove commented lines

File: jenkins/Jenkinsfile [hunk 2]
```diff
@@ -17,16 +17,16 @@ pipeline {
...
                 bash setup_miniconda.sh -p ${WORKSPACE}
                 '''
                 sh '''#!/usr/bin/env bash
                 set -e
                 source $WORKSPACE/miniconda/etc/profile.d/conda.sh
-                conda create -n cookiecutter
-                conda activate cookiecutter
-                conda install -c conda-forge cookiecutter
-                cookiecutter --no-input .
+                conda create -n copier
```
Comment (ruestefa) [inline jenkins/Jenkinsfile:26]: @regDaniel make launch jenkins to test this

File: .pre-commit-config.yaml (hunk 1 above)
Comment (ruestefa) [reply .pre-commit-config.yaml:23]: Done.

**Comment by ruestefa** [comment]
Merged and updated PR base to #48.

**Comment by regDaniel** [comment]
launch jenkins

File: jenkins/Jenkinsfile (hunk 2 above)
Comment (regDaniel) [reply jenkins/Jenkinsfile:26]: done

**Pull request closed without merging**